setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='citrination-client',
-      version='3.1.0',
+      version='3.2.0',
```

Comment:
Good catch; I didn't think about that!

Reply:
This is our usual pattern. Merge into develop without version changes then create a release branch that includes the version bump.